### DISCUSSION TITLE
How to customize animation of Tab Content without using TabPane?

### DISCUSSION THREAD

**Discussion by webdesign2be** [ant-design/ant-design#40865] [Q&A]
Hi!

Like the title states, how to customize animation of tab content without using TabPane?
I would like to use [react-spring](https://www.react-spring.dev/) to animate between changing tabs.

Is there a way to do that without using the deprecated <Tabs.TabPane /> ?

Thanks in advance for any answer!

**Comment by zombieJ** [comment]
You can HOC the Tab leave origin Tabs.TabPane empty and add div with the real content with animation lib yourself.

**Reply by webdesign2be** [reply]
Could you give me an example?
I don't understand really what you mean.
